sdk/cli/src/commands/hasura/track-table.ts

```diff
@@ -0,0 +1,158 @@
+import { missingApplication } from "@/error/missing-config-error";
+import { nothingSelectedError } from "@/error/nothing-selected-error";
+import { hasuraPrompt } from "@/prompts/cluster-service/hasura.prompt";
+import { instancePrompt } from "@/prompts/instance.prompt";
+import { serviceSpinner } from "@/spinners/service.spinner";
+import { createExamples } from "@/utils/commands/create-examples";
+import { getApplicationOrPersonalAccessToken } from "@/utils/get-app-or-personal-token";
+import { getHasuraEndpoints } from "@/utils/get-cluster-service-endpoint";
+import { Command } from "@commander-js/extra-typings";
+import { createSettleMintClient } from "@settlemint/sdk-js";
+import { loadEnv } from "@settlemint/sdk-utils/environment";
+import { intro, note, outro, spinner } from "@settlemint/sdk-utils/terminal";
+import type { DotEnv } from "@settlemint/sdk-utils/validation";
+
+export function hasuraTrackTableCommand() {
+  return new Command("track-table")
+    .alias("tt")
+    .description("Track a table in Hasura")
+    .usage(
+      createExamples([
+        {
+          description: "Track a table in the public schema",
+          command: "hasura track-table --schema public --table user",
```

Comment:
should i change this usage to be like `hasura track-table user --schema public` so `hasura track-table <table-name> --schema public`?

also i didn't add an 'untrack' table command, because it's not needed right now, but should i add that too?

Reply:
Isn't there an option to track it in all schemas and all tables?

Reply:
untrack is not needed